test1.md

```diff
@@ -0,0 +1 @@
+testing the workflow
```

Comment:
@pugnere well let's make this a proper test 😄 
Please use capital at beginning of sentence and end with a `.`

You should be able to make the changes, make a new commit and push again.